clang/lib/StaticAnalyzer/Core/BugSuppression.cpp

```diff
@@ -117,7 +117,12 @@ class CacheInitializer : public DynamicRecursiveASTVisitor {
     }
   }
 
-  CacheInitializer(Ranges &R) : Result(R) {}
+  CacheInitializer(Ranges &R) : Result(R) {
+    ShouldVisitTemplateInstantiations = true;
+    ShouldWalkTypesOfTypeLocs = false;
+    ShouldVisitImplicitCode = false;
+    ShouldVisitLambdaBody = true;
```

Comment:
These are fields right? Maybe initialize them as constructor initializers? (I kinda missed the part where they aren't virtual methods anymore.)

Reply:
These are fields of the base class. This is the only way.